packages/amplify_storage_s3/android/src/main/kotlin/com/amazonaws/amplify/amplify_storage_s3/types/FlutterRemoveRequest.kt

```diff
@@ -25,7 +25,7 @@ data class FlutterRemoveRequest(val request: Map<String, *>) {
     private fun setOptions(request: Map<String, *>): StorageRemoveOptions {
         if (request["options"] != null) {
             val optionsMap = request["options"] as Map<String, *>
-            var options: StorageRemoveOptions.Builder = StorageRemoveOptions.builder()
```

Comment:
Assuming this is just to accomodate the version bump for android library.

Reply:
yup, that was the backwards incompatible change in android library.